Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Migrations/FbMigrationsSqlGenerator.cs

```diff
@@ -247,27 +248,37 @@ protected override void Generate(RenameIndexOperation operation, IModel model, M
 
         protected override void Generate(RenameSequenceOperation operation, IModel model, MigrationCommandListBuilder builder)
```

Comment:
I had fun with this one, does it seems correct for you or do you see a simpler/better approach ?

Reply:
I thought it was good!

Reply:
Hum, yes, but no. I forgot to deal with the removal of the previous generator once the new one is created. Also, should this method be responsible of the update of the dependencies ? I mean, should be up to me to alter the stored procedures and triggers using this generator or is it up to the user to deal with it before using the RenameSequence feature ? Would be quite complicated to list, get the ddl and update those if required..

Reply:
Renaming generator/sequence is not supported in Firebird. I'm throwing an exception for such cases. Dealing with all the dependencies is not a task for this code. I'm writing also (metadata comparer for Firebird](http://www.ibphoenix.com/products/software/dbcomparer) and some of these cases are impossible to handle.